testdata/networking/ptp/namespace/45/ns.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
```

Comment:
testdata/networking/ptp/namespace/43/ns43.yaml  and testdata/networking/ptp/namespace/45/ns.yaml
Please use the same name style.

Reply:
fixed it, thanks!